DeviceWhoAmI.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$id": "http://https://github.com/harp-tech/protocol/DeviceWhoAmI.json",
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "definitions": {
+        "device": {
+            "properties": {
+                "contributor": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string",
+                    "pattern": "^[a-zA-Z][a-zA-Z0-9_]*$"
+                },
+
+                "organization": {
+                    "type": "string"
+                },
+                "projectWebsite": {
+                    "type": "string"
+                },
+                "sourceRepository": {
```

Comment:
Again for consistency with nuget I would use `repositoryUrl` here:
https://learn.microsoft.com/en-us/nuget/reference/nuspec#repository